source/PlayerInfo.cpp

```diff
@@ -1353,33 +1363,22 @@ void PlayerInfo::Land(UI *ui)
 // which case a message will be returned.
 bool PlayerInfo::TakeOff(UI *ui)
 {
-	// This can only be done while landed.
-	if(!system || !planet)
+	if(!LeavePlanet())
 		return false;
 
 	flagship = FlagshipPtr();
 	if(!flagship)
 		return false;
 
-	shouldLaunch = false;
-	Audio::Play(Audio::Get("takeoff"));
```

Comment:
Where did the takeoff sound go? This line is removed but I don't see it being readded anywhere.

Reply:
readded